test/integration/api.bats

```diff
@@ -197,9 +251,50 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker save" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	run docker_swarm pull busybox
+	[ "$status" -eq 0 ]
+
+	temp_file_name=$(mktemp)
+	# make sure busybox image exists
+	run docker_swarm images 
+	[ "$status" -eq 0 ]
+	[[ "${lines[*]}" == *"busybox"* ]]
+
+	# save >, image->tar
+	run docker_swarm save busybox > $temp_file_name
+	[ "$status" -eq 0 ]
+
+	# saved image file exists
+	[ -f $temp_file_name ]
+	# after ok, delete saved tar file
```

Comment:
Here run

`file $temp_file_name`

and verify that `$output` contains `tar archive`, just to make sure it's not garbage